planet/cli/orders.py

```diff
@@ -228,4 +229,4 @@ async def create(ctx, name, ids, bundle, item_type, email, cloudconfig, clip,
     async with orders_client(ctx) as cl:
         order = await cl.create_order(request)
 
-    echo_json(order.json, pretty)
+    echo_json(order.to_dict(), pretty)
```

Comment:
If we're encoding to JSON as late as possible this can just be an ordinary function and not a property that we might cache or whatever.

Reply:
The property `json` could be a function that calls e.g. `to_dict()`, so it seems the change would be based on whether it improves ux, right?

Reply:
Right. We're only going to call `Order.to_dict()` once in a CLI command, just before we print the JSON-encoded result, and I think that'll very likely be true in user code.

Also I think `json.dumps(order.to_dict())` makes more sense that `json.dumps(order.json)`. The second one reads like we're JSON-encoding twice.